conf/base.config

```diff
@@ -12,10 +12,8 @@ vim: syntax=groovy
  */
 
 process {
-
-  // Variable to specify the docker / singularity image tag to use
-  // Picks up on use of -r 1.3 in nextflow command
-  container = { "scilifelab/ngi-rnaseq:${workflow.revision ? workflow.revision : 'latest'}" }
```

Comment:
We need to leave this here as we need `process.container`. Can just set it to `params.container` to pick up on the variable that's been defined in `nextflow.config`.